src/main/java/io/lakefs/routerfs/PathMapper.java

```diff
@@ -199,14 +199,14 @@ public Path mapPath(Path origPath) {
     private Path convertPath(Path path, PathMapping pathMapping) {
         String str = path.toString();
         String convertedPath = str.replaceFirst(pathMapping.getSrcConfig().getPrefix(), pathMapping.getDstConfig().getPrefix());
-        LOG.trace("Converted % to % using path mapping %s", path, convertedPath,  pathMapping);
+        LOG.trace("Converted {} to {} using path mapping {}", path, convertedPath,  pathMapping);
         return new Path(convertedPath);
     }
 
     private PathMapping findAppropriatePathMapping(Path path) {
         Optional<PathMapping> appropriateMap = pathMappings.stream()
                 .filter(pm -> pm.isAppropriateMapping(path)).findFirst();
-        return appropriateMap.isPresent() ? appropriateMap.get() : null;
+        return appropriateMap.orElse(null);
```

Comment:
This is a good change -- but it raises the question why not just change the return type to `Optional<PathMapping>` and handle presence that way down below when you use it?

(Not required, of course!)